_config.yml

```
@@ -64,7 +64,7 @@ navigation:
       - title: Events
         url: https://theforeman.org/events
       - title: Community Demos
-        url: /demos/2021.html
+        url: /demos/2022.html
```

I wonder if this should start linking to the demos overview
```suggestion
        url: /demos/
```
That way users can also look back at the 2021 and earlier demos.

---

In reality, why would anyone want to watch old demos? 
On YouTube, there is a stagnation in views after about a month.
QE etc are interested for a max of about six months. 
I don't want to fetishize this process. This suggestion adds an extra step for users.

---

I did see (after I placed this comment) you added a link on the 2022 page itself. That is something we previously didn't have and may be sufficient to cover the use case.